lms/djangoapps/grades/transformer.py

```diff
@@ -64,6 +68,47 @@ def transform(self, block_structure, usage_context):
         pass
 
     @classmethod
+    def _collect_explicit_graded(cls, block_structure):
+        """
+        Collect the 'explicit_graded' field for every block.
+        """
+        def _set_field(block_key, field_value):
+            """
+            Sets the explicit graded field to the given value for the
+            given block.
+            """
+            block_structure.set_transformer_block_field(block_key, cls, cls.EXPLICIT_GRADED_FIELD_NAME, field_value)
+
+        def _get_field(block_key):
+            """
+            Gets the explicit graded field to the given value for the
+            given block.
+            """
+            return block_structure.get_transformer_block_field(block_key, cls, cls.EXPLICIT_GRADED_FIELD_NAME)
+
+        block_types_to_ignore = ('course', 'chapter', 'sequential')
```

Comment:
Also, it looks like this PR moves us further away from supporting grading aggregation at an arbitrary level.  Probably necessary here, but just want to call it out.